webpaste.el

```diff
@@ -155,7 +155,14 @@ This uses `browse-url-generic' to open URLs."
      :post-field "code"
      :post-lang-field-name "language"
      :lang-overrides ((emacs-lisp-mode . "clojure"))
-     :success-lambda webpaste--providers-success-response-url))
+     :success-lambda webpaste--providers-success-response-url)
+
+    ("bpa.st"
+     :uri "https://bpa.st/api/v1/paste"
+     :post-data (("expiry" . "1day"))
+     :post-field-lambda (lambda () #'webpaste--providers-pinnwand-request)
```

Comment:
I would like to get rid of these lambdas, it makes it look nicer, but that requires changes in several places :)
```suggestion
     :post-field-lambda webpaste--providers-pinnwand-request
```